cross/ffmpeg/Makefile

```diff
@@ -183,17 +181,19 @@ CONFIGURE_ARGS += --cpu=e500 --disable-altivec
 endif
 endif
 
+
+ifeq ($(findstring $(ARCH),$(i686_ARCHS) $(x64_ARCHS)),$(ARCH))
+BUILD_DEPENDS = native/nasm
+NASM_PATH = $(relapath $(WORK_DIR)/../../../native/nasm/work-native/install/usr/local/bin)
+ENV += PATH=$(NASM_PATH):$$PATH
+endif
```

Comment:
Have you found out why it failed for x64-6.1 and 7.0?

Reply:
finally yes (just a typo)